package.json

```diff
@@ -34,21 +34,21 @@
     "index.js"
   ],
   "engines": {
-    "node": "^14.13.1 || >=16.0.0"
+    "node": ">=18.12.0"
```

Comment:
In `stylelint-stylistic/stylelint-stylistic`, this requirement changed to `"^18.12 || >=20.9"`

https://github.com/stylelint-stylistic/stylelint-stylistic/blob/de3a29fa70fa095c38295e7f8b4a4dd38cc4f75c/package.json#L25

Reply:
I'll just follow the minimum version requirement and leave the rest out.